vite-project/src/components/Card.tsx

```diff
@@ -1,19 +1,23 @@
-import "./Card.css";
 import { useState } from "react";
-import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faHeart } from "@fortawesome/free-solid-svg-icons";
 import { FormData } from "./Form";
-
-interface FavoriteCard {
+import { Typography, Box, List } from "@mui/material";
+import FavoriteIcon from "@mui/icons-material/Favorite";
+import { useTheme } from "@mui/material";
+export interface FavoriteCard {
   name: string;
   artist: string;
 }
+import { ThemeProvider } from "@mui/material/styles";
+import { Theme } from "@mui/material/styles";
 
 interface Props {
   data: FormData;
+  Mytheme: Theme;
```

Comment:
And you dont need to pass theme through props. If you need, for example, the box with background color, that defined in your theme, you can just do `backgroundColor: theme => theme.palette.background.default` or something like that